src/NetAnalyzers/Core/Microsoft.CodeQuality.Analyzers/QualityGuidelines/MarkMembersAsStatic.cs

```diff
@@ -308,24 +319,6 @@ void Add(INamedTypeSymbol? symbol)
             return builder?.ToImmutable() ?? ImmutableArray<INamedTypeSymbol>.Empty;
         }
 
-        private static bool IsSkippedMethod(IMethodSymbol methodSymbol, WellKnownTypeProvider wellKnownTypeProvider)
```

Comment:
Is there any need to keep all the web attribute exclusions (see https://github.com/dotnet/roslyn-analyzers/pull/3875/files#diff-b71c5f4c8942242dec4740ebf6efda72R282-R298) now that we don't report on public APIs for web projects?

Reply:
Hmm @pranavkm for advice. Can these web attributes be applied on non-public methods? Unless we are sure, I'd prefer to keep the attribute based exclusions.

Reply:
The link is bugged (I can't see where it's meant to go), but in general MVC (5.0 and Core) will only discover public types and methods.

Reply:
@pranavkm Here is the link on master: https://github.com/dotnet/roslyn-analyzers/blob/master/src/NetAnalyzers/Core/Microsoft.CodeQuality.Analyzers/QualityGuidelines/MarkMembersAsStatic.cs#L270-L287

Reply:
Those are required to be on public types or methods. Feel free to remove these exclusions.